src/providers/network-state/network-state.ts

```diff
@@ -46,6 +43,9 @@ export class NetworkStateProvider {
   }
 
   public getNetworkState(): ConnectionStatus {
+    if (!this.networkStatus$) {
```

Comment:
Is this a workaround for running in browser? I don't understand the reasoning, so might be worth a comment explaining why.

Reply:
It is - and I'm going to add some JSdocs in this file to explain.

Reply:
Added jsdoc for this file.